examples/lightrunjavaagent.yaml

```diff
@@ -3,38 +3,49 @@ kind: LightrunJavaAgent
 metadata:
   name: example-cr 
 spec:
```

Comment:
@yanivshani100 Please review. Let me know if it may be explained in a more simple way